### DISCUSSION TITLE
What grid breakpoint between sm and md different?

### DISCUSSION THREAD

**Discussion by shipcake** [heroui-inc/heroui#480] [Help]
I have project use this and I have a question.

I put grid system xs and sm different value. It has nothing to change UI between 0-960px. I must design UI compatible to ipad screen size ~800px. please explain

**Comment by Prathapkarumanchi** [comment]
Please check this https://nextui.org/docs/layout/container#apis section.
Based on the screen-size that you want to support you have to apply the break point
